src/app/components/shared/Helpers/helpers.tsx

```diff
@@ -0,0 +1,25 @@
+import { themeSpacing } from "@amsterdam/asc-ui"
+import React from "react"
+import styled from "styled-components"
+export const capitalizeString = (s: string) => s.charAt(0).toUpperCase() + s.slice(1)
+
+export const appendTimeToDate = (date: string) => {
+  if (date === "") return ""
+  return `${ date }T12:00:00+0200`
+}
+
+const Ul = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  li {
+    padding: 0 0 ${ themeSpacing(1) } 0;
+    line-height: 1.15;
+  }
+`
+export const mapArrayToList = (list: any[]) =>
```

Comment:
list has type `string[]`

Reply:
Done, and made separate component for MapArrayToList